src/components/TodoList.js

```diff
@@ -0,0 +1,42 @@
+import React, { Component } from "react";
+import Todo from "./Todo.js";
+
+class TodoList extends Component {
+  constructor() {
+    super();
+
+    this.state = {
+      todos: [],
+      newTodo: ""
+    };
+  }
+
+  render() {
+    return (
+      <div>
+        {this.state.todos.map((todo, i) => <Todo key={i} todo={todo} />)}
+        <form onSubmit={this.addTodo}>
+          <input
+            type="text"
+            onChange={this.newInput}
+            placeholder="Add To List"
+            value={this.state.newTodo}
+          />
+        </form>
+      </div>
+    );
+  }
+
+  newInput = event => {
+    this.setState({ newTodo: event.target.value });
+  };
+
+  addTodo = event => {
+    const list = this.state.todos;
```

Comment:
This would be a good time to use object destructuring:

```
const { newTodo, todos } = this.state;

todos.push(newTodo);
this.setState({ newTodo: '', todos })
```